Cyborg/VectorView.swift

```diff
@@ -368,3 +435,26 @@ final class ThemeableGradientLayer: CAGradientLayer {
     }
 
 }
+
+extension CGSize {
+        
+    func scaleAspectFit(in dimensions: CGSize) -> CGSize {
+        return aspectFill(dimensions: dimensions,
+                          with: min)
+    }
+    
+    func scaleAspectFill(in dimensions: CGSize) -> CGSize {
+        return aspectFill(dimensions: dimensions,
+                          with: max)
+    }
+    
+    func aspectFill(dimensions: CGSize,
```

Comment:
Any reason this isn't prefixed with `scale` like the two functions above it?

Reply:
I wasn't sure of a good name for this actually. I wanted something that was generic, and I now realize that "Fill" isn't generic at all.